examples/Playground.hs

```diff
@@ -12,6 +13,20 @@ import qualified Data.HashMap.Strict as HashMap
 import qualified Data.List.NonEmpty as NonEmpty
 import qualified Data.Text.IO as TIO
 
+data Test = Test
+  { testB :: Bool
+  , testI :: Int
+  , testF :: Double
+  , testS :: Text
+  }
+
+testT :: BiToml Test
+testT = Test
+ <$> bool   (key ["testB"]) .= testB
+ <*> int    (key ["testI"]) .= testI
+ <*> double (key ["testF"]) .= testF
+ <*> str    (key ["testS"]) .= testS
```

Comment:
This is how it looks like. With `IsString` instance for `Key` can be even better! But it requires to add extra validation to `Key` in conversion level.